FAQ.md

```diff
@@ -0,0 +1,24 @@
+## Frequently Asked Questions
+
+- [Should I add an Xcode project to a gitignore file?](#should-i-add-an-xcode-project-to-a-gitignore-file)
```

Comment:
I'd get rid of the index because they often break when developers change titles later on and forget to update it. The list is not long so I think it's manageable for now. If the list grows a lot in the future, we can figure out a way to generate the index dynamically rather than having to maintain it statically ourselves.